getting-started.adoc

```diff
@@ -47,26 +47,6 @@ RHMAP provides a Raincatcher project template to help you run the demonstration.
 
 .. Click *Finish* at the bottom of the page when the project is created. 
 
-. Create a blank forms theme as follows:
```

Comment:
After we publish this change we will need to send email to feedhenry-raincatcher to inform users that we removed forms and documentation was updated.